values.yaml

```diff
@@ -85,7 +85,7 @@ affinity: {}
 image:
   registry: public.ecr.aws/cyral
   repository: cyral-sidecar
-  tag: v4.13.9
+  tag: ""
```

Comment:
Shouldn't we call this `version` instead? Please check what is more common (better yet the *recommended*) practice throughout Helm charts.